backend/build.gradle

```diff
@@ -1,30 +1,79 @@
 plugins {
-	id 'org.springframework.boot' version '2.3.4.RELEASE'
-	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
-	id 'java'
+    id 'org.springframework.boot' version '2.3.4.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'java'
+    id "org.openapi.generator" version "4.3.1"
```

Comment:
I replaced the tabs with spaces in the whole file. Due to our editor settings, IntelliJ will always add spaces for new entries.